habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -59,12 +60,20 @@ def _setup_actor_critic_agent(self, ppo_cfg: Config) -> None:
         """
         logger.add_filehandler(self.config.LOG_FILE)
 
-        self.actor_critic = PointNavBaselinePolicy(
-            observation_space=self.envs.observation_spaces[0],
-            action_space=self.envs.action_spaces[0],
-            hidden_size=ppo_cfg.hidden_size,
-            goal_sensor_uuid=self.config.TASK_CONFIG.TASK.GOAL_SENSOR_UUID,
-        )
+        if 'IMAGEGOAL_SENSOR' in self.config.TASK_CONFIG.TASK.SENSORS:
```

Comment:
I think there should be a cleaner way of doing it, any ideas?

Reply:
Currently no.  We need to extend our current registry to also take in policies, that way you can just specific `ImageGoalPolicy` or something in the config.